inst/unitTests/runit.xts.R

```diff
@@ -50,3 +50,52 @@ test..xts_dimnames_in_dots <- function() {
   y <- xts(1:5, index(x), dimnames = list(NULL, "x"))
   checkEquals(x, y)
 }
+
+checkXtsClass <- function(xts, class) {
```

Comment:
This test looks for attributes on specific objects, but it probably should use the extractor functions `indexClass()` and `tclass()`.